src/App.jsx

```diff
@@ -0,0 +1,72 @@
+import { useEffect, useState } from 'react';
+import './App.css';
+import destinationsdata from "./destination.json"
```

Comment:
be consistent with quotes usage, either ' or " but not both
in jsx for all that is javascript ' and for html " (for the import you choose)
Point 19:
https://rjroopal.medium.com/best-practices-for-clean-react-apps-5a891cbe48ab